diff --git a/node_modules/@agoric/rpc/dist/index.js b/node_modules/@agoric/rpc/dist/index.js
index 06d72bd..213bc99 100644
--- a/node_modules/@agoric/rpc/dist/index.js
+++ b/node_modules/@agoric/rpc/dist/index.js
@@ -1,3 +1,3 @@
 /* eslint-disable import/extensions */
-export * from './src/chainStorageWatcher';
-export * from './src/types';
+export * from './src/chainStorageWatcher.js';
+export * from './src/types.js';
diff --git a/node_modules/@agoric/rpc/dist/src/batchQuery.js b/node_modules/@agoric/rpc/dist/src/batchQuery.js
index 910dd87..a73c927 100644
--- a/node_modules/@agoric/rpc/dist/src/batchQuery.js
+++ b/node_modules/@agoric/rpc/dist/src/batchQuery.js
@@ -1,4 +1,4 @@
-import { AgoricChainStoragePathKind } from './types';
+import { AgoricChainStoragePathKind } from './types.js';
 export const pathToKey = (path) => path.join('.');
 export const keyToPath = (key) => {
     const parts = key.split('.');
diff --git a/node_modules/@agoric/rpc/dist/src/chainStorageWatcher.js b/node_modules/@agoric/rpc/dist/src/chainStorageWatcher.js
index 02a8c44..b81e441 100644
--- a/node_modules/@agoric/rpc/dist/src/chainStorageWatcher.js
+++ b/node_modules/@agoric/rpc/dist/src/chainStorageWatcher.js
@@ -1,8 +1,8 @@
 /* eslint-disable no-use-before-define */
 /* eslint-disable import/extensions */
-import { makeClientMarshaller } from './marshal';
-import { AgoricChainStoragePathKind } from './types';
-import { batchVstorageQuery, keyToPath, pathToKey } from './batchQuery';
+import { makeClientMarshaller } from './marshal.js';
+import { AgoricChainStoragePathKind } from './types.js';
+import { batchVstorageQuery, keyToPath, pathToKey } from './batchQuery.js';
 const defaults = {
     newPathQueryDelayMs: 20,
     refreshLowerBoundMs: 2000,
